tests/network-tests/package.json

```diff
@@ -4,7 +4,7 @@
   "license": "GPL-3.0-only",
   "scripts": {
     "build": "tsc --build tsconfig.json",
-    "test": "tap --files ts-node/register src/constantinople/tests/proposals/*Test.ts",
+    "test": "tap --files ts-node/register src/constantinople/tests/proposals/*Test.ts -T",
```

Comment:
what does the `-T` argument do?

Reply:
`-T` removes timeout. Default timeout may run out during application startup. The actual timeout is set up in the beginning of each testing scenario.